### ISSUE TITLE
Fix github actions

### ISSUE THREAD

**Issue by draxaris1010** [ZeusWPI/zeus.ugent.be#679]
There might be something wrong with [bulma](https://sass-lang.com/documentation/breaking-changes/color-functions/)
See the action runs on d2a2582fbaba4a01ab6834b9cea0a48fff217999
```
Sass::CompileError: Undefined function.

Error: Undefined function.
    ╷
108 │     "red": color.channel($color, "red", $space: rgb),
    │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  /home/runner/work/zeus.ugent.be/zeus.ugent.be/content/assets/stylesheets/bulma/sass/utilities/functions.scss 108:12         bulmaColorLuminance()
  /home/runner/work/zeus.ugent.be/zeus.ugent.be/content/assets/stylesheets/bulma/sass/utilities/functions.scss 137:7          bulmaFindColorInvert()
  /home/runner/work/zeus.ugent.be/zeus.ugent.be/content/assets/stylesheets/bulma/sass/utilities/derived-variables.scss 16:15  @forward
  /home/runner/work/zeus.ugent.be/zeus.ugent.be/content/assets/stylesheets/bulma/sass/utilities/_index.scss 6:1               @forward
  bulma/_index.scss 1:1                                                                                                       @use
  - 46:1                                                                                                                      root stylesheet

  0. /nix/store/cjrvinhsk987livlprkgb2j290yrhi45-ruby3.3-sass-embedded-1.79.3/lib/ruby/gems/3.3.0/gems/sass-embedded-1.79.3/lib/sass/compiler/host.rb:86:in `compile_request'
  1. /nix/store/cjrvinhsk987livlprkgb2j290yrhi45-ruby3.3-sass-embedded-1.79.3/lib/ruby/gems/3.3.0/gems/sass-embedded-1.79.3/lib/sass/compiler.rb:171:in `compile_string'
  2. /nix/store/cjrvinhsk987livlprkgb2j290yrhi45-ruby3.3-sass-embedded-1.79.3/lib/ruby/gems/3.3.0/gems/sass-embedded-1.79.3/lib/sass/embedded.rb:37:in `compile_string'
  3. lib/filters/dart_sass_custom.rb:18:in `run'
  4. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/filter.rb:188:in `setup_and_run'
  5. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/executor.rb:22:in `filter'
  6. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/compilation_phases/recalculate.rb:32:in `block in run'
  7. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/action_sequence.rb:48:in `each'
  8. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/action_sequence.rb:48:in `each'
  9. /nix/store/k1xrgfsbq7lxfyfzbxrprhx4djgnbry1-ruby3.3-nanoc-core-4.13.0/lib/ruby/gems/3.3.0/gems/nanoc-core-4.13.0/lib/nanoc/core/compilation_phases/recalculate.rb:29:in `run'
  ... 6 lines omitted (see crash.log for details)
```
https://github.com/ZeusWPI/zeus.ugent.be/actions/runs/11748071814/job/33062364860#step:3:157

**Comment by draxaris1010** [comment]
The site does however build correctly without errors on my machine

**Comment by xerbalind** [comment]
Did you update the submodules, particularly the Bulma one? The build action starts with updating those.

**Comment by draxaris1010** [comment]
I've updated the submodule and now it fails with the same error

**Comment by xerbalind** [comment]
Jup tried it on my machine, also failed.
Updating the dependencies helped see pr #677 on my machine. But it still fails with the build action. Also updated the `gemset.nix` and cleared the cache.

**Comment by xerbalind** [comment]
fixed by #677
